express-intro/server.ts

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+
+const app = express();
+app.listen(8080, () => {
```

Comment:
It is customary to put this at the bottom of the file. That way, the code reads "First, configure and set up the server, then start listening".

It works with `app.listen` anywhere, though.